components/EditCarForm/index.js

```diff
@@ -53,7 +53,13 @@ const groups = [
     ],
   },
 ];
-
+function handleUploadFile(event) {
+  event.preventDefault();
+  console.log(event.target.files[0].size);
```

Comment:
Please delete logs for developing purposes

Reply:
Done!